master/usage/troubleshooting/index.md

```diff
@@ -67,27 +81,50 @@ No IPv6 peers found.
 
 If you do not see this, please check the following.
 
-- Can your hosts ping each other?  There must be IP connectivity between the
-  hosts.
+- Make sure there is IP connectivity between your hosts.
+
+- Make sure your network allows the requisite BGP traffic on TCP port 179.
+
+### Disable Ubuntu (or GNOME) NetworkManager
+
+Disable [NetworkManager](https://help.ubuntu.com/community/NetworkManager) before
+attempting to use {{site.prodname}} networking.
+
+NetworkManager manipulates the routing table for interfaces in the default network
+namespace where {{site.prodname}} veth pairs are anchored for connections to containers.
+This can interfere with the {{site.prodname}} agent's ability to route correctly.
+
+You can configure interfaces in the `/etc/network/interfaces` file if the
+NetworkManager removes your host's interfaces. See the Debian
+[NetworkConfiguration](https://wiki.debian.org/NetworkConfiguration)
+guide for more information.
+
+## Running `sudo calicoctl ...` with Environment Variables
 
-- Your hosts' names must be different. {{site.prodname}} uses hostname as a key in the
-  etcd data, and the etcd data is used to autogenerate the correct BIRD
-  config - so a duplicate hostname will prevent correct BIRD setup.
+If you use `sudo` for commands like `calicoctl node run`, remember that your environment
+variables will not be transferred to the `sudo` environment.  You can run `sudo` with
+the `-E` flag to include your environment variables:
 
-- There must not be iptables rules, or any kind of firewall, preventing
-  communication between the hosts on TCP port 179.  (179 is the BGP port.)
+```shell
+    sudo -E calicoctl node run
```

Comment:
nit: it would be better if these sudos lined up with the backticks...right now, they are indented in the rendered output and it just looks a little off. Not a big deal at all but something I observed.